gateleen-hook/src/main/java/org/swisspush/gateleen/hook/HookHandler.java

```diff
@@ -1032,23 +1041,33 @@ private void handleListenerRegistration(final HttpServerRequest request) {
             final String listenerStorageUri = hookRootUri + HOOK_LISTENER_STORAGE_PATH + getUniqueListenerId(request.uri());
 
             // Extract expireAfter from the registration header.
-            final String myExpireAfter = request.headers().get( X_EXPIRE_AFTER );
             final String expirationTime;
-            if( "-1".equals(myExpireAfter) ){
-                expirationTime = EXPIRATION_TIME_NEVER;
-            }else{
-                Integer expireAfter = ExpiryCheckHandler.getExpireAfter(request.headers());
-                if (expireAfter == null) {
+
+            { // Evaluate expiration time
+                Integer expireAfter;
+                try{
+                    expireAfter = ExpiryCheckHandler.getExpireAfter2( request.headers() );
+                    // Use default if header is not set.
+                    if( expireAfter == null ) expireAfter = DEFAULT_HOOK_STORAGE_EXPIRE_AFTER_TIME;
```

Comment:
To keep a continuous code style across the whole framework, avoid using _single line if statements_

Reply:
Oops. Seems valid. 👍